lnwallet/chainfee/estimator.go

```diff
@@ -250,7 +260,20 @@ func (b *BtcdEstimator) EstimateFeePerKW(numBlocks uint32) (SatPerKWeight, error
 //
 // NOTE: This method is part of the Estimator interface.
 func (b *BtcdEstimator) RelayFeePerKW() SatPerKWeight {
-	return b.minFeeManager.fetchMinFee()
+	// Compare the minimum relay fee against the filterManager's suggested
+	// feerate and return the higher of the two.
+	minFee := b.minFeeManager.fetchMinFee()
+
+	filterFee, err := b.filterManager.fetchMedianFilter()
+	if err == errNoFilters {
+		return minFee
+	}
+
+	if filterFee > minFee {
```

Comment:
Hmm, I wonder if we want to allow the user to override this if they _really_ want to publish at that low fee. I guess ultimately, they can specify the `sat_per_vbyte` field in that case.